compiler/compiler.py

```diff
@@ -32,47 +33,38 @@ def compile_upload(upload_id: str, output_format: str='pdf',
     output_format: str
         The desired output format. Default: "pdf". Other potential values:
         "dvi", "html", "ps"
-    fms_endpoint: str
-        The API endpoint for the FileManagementService.
     preferred_compiler: str
         The preferred tex compiler for use with the source package.
-    output_endpoint: str
-        The API endpoint for uploading compiled files.
     """
-
-    if fms_endpoint is None:
-        fms_endpoint = current_app.config['FMS_ENDPOINT']
-
-    if output_endpoint is None:
-        output_endpoint = current_app.config['OUTPUT_ENDPOINT']
-
-
-    # 1. Retrieve the source package
-    fms = FileManagementService(fms_endpoint)
-
-    body, headers = fms.get_upload_content(upload_id)
-    etag = headers['ETag']
+    source_package = filemanager.get_upload_content(upload_id)
```

Comment:
What if there is a problem getting the content? See https://github.com/cul-it/arxiv-compiler/blob/master/compiler/services/filemanager.py#L33 for a bunch of possible exceptions that you might encounter here.